src/main/java/com/manoelcampos/javadoc/coverage/stats/ClassDocStats.java

```diff
@@ -46,41 +45,46 @@
     private List<MethodDocStats> methodsStats;
     private List<MethodDocStats> constructorsStats;
 
-    public ClassDocStats(final ClassDoc doc) {
+    public ClassDocStats(final ClassDoc doc, Configuration config) {
         this.doc = doc;
```

Comment:
The config parameter should be stored in a config attribute. This way, you avoid passing the config around several methods of the class.